web/src/narrow_title.ts

```diff
@@ -21,6 +23,9 @@ export function compute_narrow_title(filter) {
         if (inbox_util.is_visible()) {
             return $t({defaultMessage: "Inbox"});
         }
+        // TODO(evy): should we fall back on something?
+        // it seems like we never expect to be here, should we warn?
+        return "";
```

Comment:
If we had fallen through here, we would previously have thrown an error on `filter.get_title()`, which we haven’t been seeing in Sentry. So we can safely throw an error explicitly, or change the previous `if` to `assert`.

Reply:
Yeah that makes sense -- I think I've been underusing `assert`, and started using it more liberally today in newer work. Thanks!